http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/SamDAO.scala

```diff
@@ -70,7 +70,19 @@ trait SamDAO[F[_]] {
 
   /** Creates a Sam resource R with the provided google project as the parent resource using a GCP pet credential
      for the given email/project. */
-  def createResourceWithParent[R](resource: R, creatorEmail: WorkbenchEmail, googleProject: GoogleProject)(implicit
+  def createResourceWithGcpParent[R](resource: R, creatorEmail: WorkbenchEmail, googleProject: GoogleProject)(implicit
```

Comment:
Nit - wonder if `createResourceWithGoogleProjectParent` is slightly more descriptive.